src/lib/visualization/abstract_visualizer.hpp

```diff
@@ -87,16 +87,21 @@ class AbstractVisualizer {
 
   virtual ~AbstractVisualizer() = default;
 
-  void visualize(const GraphBase& graph_base, const std::string& graph_filename, const std::string& img_filename) {
+  void visualize(const GraphBase& graph_base, const std::string& img_filename) {
     _build_graph(graph_base);
-    std::ofstream file(graph_filename);
+
+    char* tmpname = strdup("/tmp/hyrise_viz_XXXXXX");
+    auto ret = mkstemp(tmpname);
```

Comment:
abbreviation ;)

Reply:
Consistent with `console.cpp`. `return` is not an option.

Reply:
kk, just wanted to be a pain.